com.unity.render-pipelines.universal/Runtime/2D/Passes/Utility/RendererLighting.cs

```diff
@@ -267,10 +267,11 @@ private static void RenderLightSet(IRenderPass2D pass, RenderingData renderingDa
             }
         }
 
-        public static void RenderLightVolumes(this IRenderPass2D pass, RenderingData renderingData, CommandBuffer cmd, int layerToRender, int endLayerValue, RenderTargetIdentifier renderTexture, RenderTargetIdentifier depthTexture, List<Light2D> lights)
+        public static void RenderLightVolumes(this IRenderPass2D pass, RenderingData renderingData, CommandBuffer cmd, int layerToRender, int endLayerValue,
```

Comment:
This will be a public API breakage? Maybe that's ok if the 2D renderer is still experimental, but otherwise something to note.

Reply:
Actually, the class is internal, even if the method is public.